addons/website/static/src/scss/website.editor.ui.scss

```diff
@@ -60,3 +60,54 @@ $o-we-switch-inactive-color: #F7F7F7 !default;
         }
     }
 }
+
+.o_we_slider_tint input[type="range"] {
+    background: linear-gradient(to right, #F00 0%, #FF0 16.66%, #0F0 33.33%, #0FF 50%, #00F 66.66%, #F0F 83.33%, #F00 100%);
+    height: 10px !important;
+    cursor: pointer;
+
+    &::-webkit-slider-runnable-track {
+        background-color: transparent !important;
+    }
+    &::-moz-range-track {
+        background-color: transparent !important;
+    }
+    &::-moz-range-progress {
+        background-color: transparent !important;
+    }
+
+    &::-webkit-slider-thumb {
+        appearance: none !important;
+        height: 20px !important;
+        border: 1px solid $o-we-bg-dark !important;
+        box-shadow: inset 0 0 0 1px white !important;
+        background: transparent !important;
+        border-radius: 0 !important;
+        margin-top: -5% !important;
+    }
+    &::-moz-range-thumb {
+        appearance: none !important;
+        height: 20px !important;
+        border: 1px solid $o-we-bg-dark !important;
+        box-shadow: inset 0 0 0 1px white !important;
+        background: transparent !important;
+        border-radius: 0 !important;
+        margin-top: -5% !important;
+    }
+}
```

Comment:
I am pretty sure there is code to share the colorpalette slider here ;) Don't just copy/paste code, always try to reuse/share it.